deployment/ansible/roles/cac-tripplanner.app/tasks/jslibs.yml

```diff
@@ -11,24 +11,29 @@
        state=present
 
 - name: Install NPM dependencies
-  npm: path="{{ gulp_dir }}" state=present
+  npm: path="{{ root_src_dir }}" state=present
```

Comment:
This may explain some problems I'm having on Jenkins right now, I'll look into switching it.